### PR TITLE
Celo staking LLM - withdraw unlocked funds

### PR DESCRIPTION
**Pull request by marco-figment**
### 📝 Description

Adds functionality to withdraw unlocked funds in Celo LLM.

### ❓ Context

- **Impacted projects**: `ledger-live-mobile`
- **Linked resource(s)**: `n/a`

### ✅ Checklist

- [✅ ] **Test coverage** Manual test + video
- [ ✅] **Atomic delivery**
- [ ✅] **No breaking changes**

### 📸 Demo

https://user-images.githubusercontent.com/98154129/188891319-281feb30-9a56-42a3-a733-cc853efe3b9c.mov

### CODE REVIEW

File: apps/ledger-live-mobile/src/families/celo/WithdrawFlow/WithdrawAmount.tsx [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+    fontWeight={highlighted ? "bold" : "semiBold"}
+    style={[styles.summaryWords, style]}
+    color={highlighted ? "live" : "smoke"}
+  >
+    {children}
+  </Text>
+);
+
+
+const Selectable = ({
```
Comment (hedi-edelbloute) [inline apps/ledger-live-mobile/src/families/celo/WithdrawFlow/WithdrawAmount.tsx:249]: Can we try to mutualize this component with the one in celo/components directory ?

**Review by haammar-ledger — CHANGES_REQUESTED** [review]
Can you please also check [my comment](https://github.com/LedgerHQ/ledger-live/pull/1071#discussion_r957522747) on the previous PR?

File: apps/ledger-live-mobile/src/families/celo/WithdrawFlow/WithdrawAmount.tsx [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+  }
+
+  const [infoModalOpen, setInfoModalOpen] = useState(false);
+
+  const openModal = useCallback(time => setInfoModalOpen(time), [
+    setInfoModalOpen,
+  ]);
+  const closeModal = useCallback(() => setInfoModalOpen(false), [
+    setInfoModalOpen,
+  ]);
```
[the remark this answers is not in the document]
Comment (marco-figment) [reply apps/ledger-live-mobile/src/families/celo/WithdrawFlow/WithdrawAmount.tsx:117]: fixed

File: apps/ledger-live-mobile/src/families/celo/WithdrawFlow/WithdrawAmount.tsx [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+import { getMainAccount } from "@ledgerhq/live-common/account/helpers";
+import { getAccountBridge } from "@ledgerhq/live-common/bridge/index";
+import { accountScreenSelector } from "../../../reducers/accounts";
+import { ScreenName } from "../../../const";
+import { TrackScreen } from "../../../analytics";
+import { formatCurrencyUnit } from "@ledgerhq/live-common/currencies/index";
+import Button from "../../../components/Button";
+import Touchable from "../../../components/Touchable";
+import SendRowsFee from "../SendRowsFee";
+import { useDebounce } from "@ledgerhq/live-common/lib/hooks/useDebounce";
```
[the remark this answers is not in the document]
Comment (marco-figment) [reply apps/ledger-live-mobile/src/families/celo/WithdrawFlow/WithdrawAmount.tsx:27]: done, except for live-common/lib/currencies, which seems to still require it.

**Comment by marco-figment** [comment]
Addressed comments.

**Comment by haammar-ledger** [comment]
@marco-figment There seems to be a lot of lint issues, mostly related to formatting I think.
If you don't do it already, the easiest is to setup your dev IDE to use the `.prettierrc` configs that we provide on each project, you can even set it up to apply the formatting automatically when saving files. :ok_hand:

**Comment by henrily-ledger** [comment]
LGTM

**Pull request merged**